src/renderer/views/dashboard/Dashboard.tsx

```diff
@@ -1,63 +1,26 @@
+import type { FC } from "react";
 import React from "react";
 
-import { Layout } from "../../components/common/layout/Layout";
 import style from "./Dashboard.module.scss";
-import type { DashboardAnalyticsInfosProps } from "./DashboardAnalyticsInfos";
-import { DashboardAnalyticsInfos } from "./DashboardAnalyticsInfos";
-import { DashboardGlobalInfos } from "./DashboardGlobalInfos";
+import { DashboardActions } from "./DashboardActions";
+import { DashboardImpact } from "./DashboardImpact";
+import { DashboardInformations } from "./DashboardInformations";
+import { DashboardRecap } from "./DashboardRecap";
+import { DashboardViewer } from "./DashboardViewer";
 
-// TODO: Fake data - to replace by PST extracted infos
-const PSTAnalyticsDatas: DashboardAnalyticsInfosProps[] = [
-    {
-        datas: [
-            {
-                numbers: [30, 50, 40],
-                title: "Messages Reçus",
-            },
-            {
-                numbers: [30, 50, 40],
-                title: "Messages Envoyés",
-            },
-            {
-                numbers: [30, 50, 40],
-                title: "Messages Supprimés",
-            },
-        ],
-        labels: ["Catégorie", "Pourcentage", "Mails", "Pièces Jointes"],
-        title: "Mails",
-    },
-    {
-        datas: [
-            {
-                numbers: [30, 50],
-                title: "Contact",
-            },
-        ],
-        labels: ["Catégorie", "Nombre de contacts", "Listes"],
-        title: "Contact",
-    },
-    {
-        datas: [
-            {
-                numbers: [30, 50, 70],
-                title: "Agenda",
-            },
-        ],
-        labels: ["Catégorie", "Pourcentage", "Mail", "Pièces Jointes"],
-        title: "Agenda",
-    },
-];
-
-export const Dashboard: React.FC = () => {
+export const Dashboard: FC = () => {
```

Comment:
Why use `FC` as is if you already have `React` object usable?
```suggestion
export const Dashboard: React.FC = () => {
```

Reply:
Readability but fair enough to use React.FC